problems/water/water.adoc

```diff
@@ -52,18 +52,23 @@ check50 cs50/2017/x/water
 ----
 
 == Staff Solution
-
+Not sure how your solution should behave?
+Run our example by calling '~cs50/pset1/water'. Assumed that the underlined text is what some user has typed.
```

Comment:
I would maybe condense these two lines into "You can run our solution to the problem by typing '~cs50/pset1/water' into the terminal" or similar.

Reply:
Making it shorter would be more appropriate